libs/extensions/std/exec/src/http-extractor-executor.ts

```diff
@@ -43,14 +46,39 @@ export class HttpExtractorExecutor extends AbstractBlockExecutor<
     context: ExecutionContext,
   ): Promise<R.Result<BinaryFile>> {
     const url = context.getPropertyValue('url', PrimitiveValuetypes.Text);
-
-    const file = await this.fetchRawDataAsFile(url, context);
-
-    if (R.isErr(file)) {
-      return file;
+    const retries = context.getPropertyValue(
+      'retries',
+      PrimitiveValuetypes.Integer,
+    );
+    const retryBackoff = context.getPropertyValue(
+      'retryBackoff',
```

Comment:
I am a big fan in including expected units in variable names that are not obvious. Id suggest `retryBackoffMS` or better `retryBackoffMicroseconds`.

Reply:
With the comment about the exponential backoff strategy, does it make sense to remove this prop completely?

Reply:
We could also go with just exponential backoffs, 1, 2, 4 etc seconds. But might as well keep both configurable, right? Might be useful if we add a prop to change backoff strategy in the future.